servicetest/conftest.py

```diff
@@ -32,21 +31,32 @@
 os.sys.path.insert(0, package_root)
 
 
-# TODO: dbus-launch needs to be cleaned up during teardown
 @pytest.fixture(scope="module")
 def dbus_launch():
     """ Setting up dbus environement variables for session bus """
-    p = subprocess.Popen('dbus-launch', shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    p = subprocess.Popen('dbus-launch', stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
     for var in p.stdout:
         sp = var.split('=', 1)
-        os.environ[sp[0]] = sp[1][:-1]
+        env_var = sp[0]
+        env_val = sp[1][:-1]
+        os.environ[env_var] = env_val
+    yield
+    # The DBUS_SESSION_BUS_PID is set to sp[1] in the last loop iteration above...
+    os.kill(int(sp[1]), 9)
```

Comment:
I think we should use the singal module instead of magic numbers for SIGKILL, SIGTERM etc...
https://docs.python.org/2/library/signal.html